README.md

```diff
@@ -1,161 +1,157 @@
-services.
-
-### On Google Cloud Platform
-
-If you are running this client on Google Cloud Platform, we handle
-authentication for you with no configuration. You just need to make sure that
-when you [set up the GCE instance][gce-how-to], you add the correct scopes for
-the APIs you want to access.
-
-``` js
-var gcs = require('@google-cloud/storage')();
-// ...you're good to go!
-```
-
-### Elsewhere
-
-If you are not running this client on Google Cloud Platform, you need a
-Google Developers service account. To create a service account:
-
-1. Visit the [Google Developers Console][dev-console].
-2. Create a new project or click on an existing project.
-3. Navigate to  **APIs & auth** > **APIs section** and turn on the following
-   APIs (you may need to enable billing in order to use these services):
-    * Google Cloud Storage
-    * Google Cloud Storage JSON API
-4. Navigate to **APIs & auth** >  **Credentials** and then:
-    * If you want to use a new service account key, click on
-      **Create credentials** and select **Service account key**. After the
-      account key is created, you will be prompted to download the JSON key
-      file that the library uses to authenticate your requests.
-    * If you want to generate a new service account key for an existing
-      service account, click on **Generate new JSON key** and download the
-      JSON key file.
-
-``` js
-var projectId = process.env.GCLOUD_PROJECT; // e.g. 'grape-spaceship-123'
-
-var gcs = require('@google-cloud/storage')({
-  projectId: projectId,
-
-  // The path to your key file:
-  keyFilename: '/path/to/keyfile.json',
-
-  // Or the contents of the key file:
-  credentials: require('./path/to/keyfile.json'),
-});
-
-// ...you're good to go!
-```
-
-
-[google-cloud]: https://github.com/GoogleCloudPlatform/google-cloud-node/
-[gce-how-to]: https://cloud.google.com/compute/docs/authentication#using
-[dev-console]: https://console.developers.google.com/project
-[node-storage-docs]: https://googlecloudplatform.github.io/google-cloud-node/#/docs/storage
-[cloud-storage-docs]: https://cloud.google.com/storage/docs/overview
+### Before you begin
+
+1.  Select or create a Cloud Platform project.
+
+    [Go to the projects page][console]
+
+1.  Enable billing for your project.
+
+    [Enable billing][billing]
+
+1.  Enable the Google Cloud Storage API.
+
+    [Enable the API][enable_api]
+
+1.  [Set up authentication with a service account][auth] so you can access the
+    API from your local workstation.
+
+[console]: https://console.cloud.google.com/
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+[enable_api]: https://console.cloud.google.com/flows/enableapi?apiid=storage-api.googleapis.com
+[auth]: https://cloud.google.com/docs/authentication/getting-started
+
+### Installing the client library
+
+    npm install --save @google-cloud/storage
+
+### Using the client library
+
+    // Imports the Google Cloud client library
```

Comment:
I think this file was generated in some form-- if that's the case, instead of using indents here, is it possible to have the generator wrap the code blocks with ticks, and in this case, ```js so that it's syntax highlighted?

Reply:
Will do: GoogleCloudPlatform/nodejs-repo-tools#23